.github/workflows/release.yml

```diff
@@ -39,6 +39,7 @@ jobs:
 
   release:
     if: github.ref == 'refs/heads/main'
+    needs: [test, e2e, e2e-eu]
```

Comment:
@sanderblue 
The release doesn't depend on those to complete first, they can be done in parallel.

Reply:
The release definitely depends on tests passing. That's how we prevent releasing most of our bugs.

Reply:
I removed the `e2e` and `e2e-eu` dependency from here to speed things up, but we need to keep `test` because we shouldn't release if `test` fails.